draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -620,6 +620,27 @@ the server.
 Clients MUST validate the received values to mitigate those risks.
 
 
+# Privacy Considerations
+
+RateLimit fields can be used to implement client tracking techniques:
```

Comment:
This is not about tracking as much as it implies reidentification.

A client that acts upon a request to rate limit is potentially reidentifiable because they react to information that might only be given to them.

It's good that you call out Retry-After as another example.  But we are generally comfortable with both as simply retrying a request does not improve the information a server receives about a client.  It only changes if the client adjusts its request in the intervening time, which might give the server more information about the client.  (Note that a client naturally reveals information to a server when it makes a request; we generally don't concern ourselves about that either.)  This is no different than in cases of connection reuse, so it probably isn't worth spending a lot of text on.

Where this gets interesting is in the way that the *server's* actions might reveal information to clients about the activity of *other clients*.

Reply:
Thanks Martin. This last point should be addressed by https://github.com/ietf-wg-httpapi/ratelimit-headers/blob/main/draft-ietf-httpapi-ratelimit-headers.md#information-disclosure-sec-information-disclosure